gremlin-test/src/main/java/org/apache/tinkerpop/gremlin/process/traversal/step/sideEffect/InjectTest.java

```diff
@@ -114,6 +115,7 @@ public void g_injectXnull_1_3_nullX() {
     }
 
     @Test
+    @Ignore
```

Comment:
the issue with these tests was with groovy execution right? i thought that was solved earlier with the changes to the translator? can these be enabled again?